src/controllers/error.controller.ts

```diff
@@ -3,7 +3,7 @@ import logger from "../utils/logger";
 import { Templates } from "../types/template.paths";
 
 const pageNotFound = (req: Request, res: Response) => {
-  return res.status(404).render(Templates.ERROR_404);
+  return res.status(404).render(Templates.ERROR_404, {templateName: Templates.ERROR_404});
```

Comment:
have you linted this pr?
It's very minor but i would have thought it would pick up there's no spacing between the curlies